src/Npgsql.NodaTime/Internal/TimestampTzMultirangeHandler.cs

```diff
@@ -1,202 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
-using NodaTime;
-using Npgsql.BackendMessages;
-using Npgsql.Internal;
-using Npgsql.Internal.TypeHandlers;
-using Npgsql.Internal.TypeHandling;
-using Npgsql.PostgresTypes;
-using NpgsqlTypes;
-
-namespace Npgsql.NodaTime.Internal;
-
-public partial class TimestampTzMultirangeHandler : MultirangeHandler<Instant>,
```

Comment:
I'm wondering, why did we never have automatic composition to multiranges, just like arrays?

Reply:
Yeah, good question, not sure.